templates/home.html

```diff
@@ -2,7 +2,8 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>BGGC Random Game</title>
+    <title>BGG | Home</title>
+    <link rel="icon" type="image/x-icon" href="http://www.boardgamegeek.com/favicon.ico">
```

Comment:
This URL actually resolves to `https://cf.geekdo-static.com/icons/favicon2.ico` so you might want to use that directly

Reply:
Ahhhhh I see!